### DISCUSSION TITLE
How do do I make Seq copy-deployable

### DISCUSSION THREAD

**Discussion by bitbonk** [datalust/seq-tickets#1262] [Help] [answered]
What is the official or best way to make Seq copy-deployable on Windows? How can I run the Seq server on Windows without having to run any installers?

**Comment by nblumhardt** [comment, marked as answer]
Hi @bitbonk; it's possible to simply copy `C:\Program Files\Seq` to another location, and run `Seq.exe` from it (with `Seq.exe service install` etc. all available). We don't generally encourage doing it, since the upgrade experience is not great (stop the service, delete everything, overwrite), and we can't guarantee it will always work in the same way, but if you have no option to run the MSI then it can work.

Another possibility is to automate MSI installation with `msiexec /i Seq-*.msi` - this runs the installer but without requiring user interactivity. Let me know if you need more info on this.

Hope this helps!
Nick

**Reply by bitbonk** [reply]
@nblumhardt Coming back to this after a while. Is you answer still true for the current version of Seq (2022.1)?

**Reply by liammclennan** [reply]
Hi @bitbonk,

Nick's advice is still correct. 

You can easily start a temporary Seq using Docker. I don't use Windows, but presumably that would work with docker desktop. Something like: 

`docker run -p 5344:80 -e ACCEPT_EULA=Y datalust/seq:latest`
